source/get-started/pojo-qs.txt

```diff
@@ -0,0 +1,345 @@
+.. _javars-pojo-qs:
+
+===========================
+Quick Start (POJO Examples)
+===========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This guide follows from the :ref:`Quick Start guide
```

Comment:
Q: what does "follows from" mean here? It's a continuation of the other quick start, or follows the same structure but uses POJOs?

Reply:
changed language